analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -1061,7 +1061,7 @@ static void Main(string[] args)
                     continue;
                 }
 
-                var _ = providerCourse.Items.Where(items => items == "item1"); // FIXME Non-compliant FP
+                var _ = providerCourse.Items.Where(items => items == "item1"); // Noncompliant FP
```

Comment:
Can you mark where the issue is raised? Or do `providerCourse.Items.ToList()` to get rid of the confusing lambda.

Reply:
I'll do both. One to minimize the case. Other to be sure we don't do anything from the `Items?` stuff on line 1058